lib/resouces/assets.dart

```diff
@@ -0,0 +1,6 @@
+class Assets {
+  const Assets._();
+
+  static const String bgLogin = "assets/images/bg_login.jpg";
+  static const String icNimbleLogo = "assets/icons/ic_nimble_logo.svg";
```

Comment:
won't you want to use this? 🥇  
https://pub.dev/packages/flutter_gen

Reply:
cool package, let me update 👌